ui/DesignSystem/Component/SourceBrowser/FileDiff.svelte

```diff
@@ -2,6 +2,39 @@
   import { Title, Icon } from "../../Primitive";
 
   export let file = null;
+
+  const lineNumL = line => {
```

Comment:
Nit: I'd prefer non-abbreviated naming in the UI code.

Eg.: `lineNumL` -> `lineNumberLeft`, `lineNum` -> `lineNumber`, etc.